<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
  <head>
    <title>Do you have a moment to talk about SNPs?</title>
    <meta charset="utf-8" />
    <meta name="author" content="James Morgan" />
    <meta name="date" content="2019-06-19" />
    <link rel="stylesheet" href="static/stylesheets/custom.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Do you have a moment to talk about SNPs?
## Translating Polygenic Risk Assessment to the clinic
### James Morgan
### 2019-06-19

---






.toc[
# The story so far...

## What's this all about?
#### Elevator Pitch (and a few definitions)
#### Project Summary
&lt;!-- #### What is a SNP? --&gt;
&lt;!-- #### Who is this "James" guy anyway?  --&gt;

## Which SNPs and Why?
### Trawling through the GWAS Catalog 

## Polygen 
### How are you going to do this anyway?
]

---

class: inverse

# Elevator Pitch

### Problem: 
&gt; &amp;raquo; Billions of dollars worth of burden to the health system.

- Annual cancer burden *~65,000 diagnoses, ~12,500 deaths*
  + Breast, Colorectal &amp; Prostate cancer + Melanoma
- Current estimation of cancer risk methods are Blunt/Inadequate 
  + Most affected individuals have no significant FHx
  


### Proposed Solution: 
&gt; &amp;raquo; Personalised Genomic Risk Assessment Platform

- Roadmap to a cost-effective reduction in cancer-related disease burden.

- Risk/Evidence-based screening
  + Identify high-risk individuals through **SNP Genotyping**
  + Precise estimation &amp; standard interpretation of &lt;br&gt;**Polygenic Risk (PRS)**

---

# What are these acronyms?

### SNP - Single Nucleotide Polymorphism

- Common genomic variation contributes to cancer risk.

- Individual SNPs contribute a near-negiligible risk.

- Combined risk-contribution may be greater than any other single factor.


### PRS - Polygenic Risk Score 

- Quantifies combined SNP-associated risk

- Various derivation methods

- Demonstrated ability to improve performance of existing risk assessment methods


---
class: inverse

# VCCC Precision Medicine Grant 

&gt; The development of a VCCC platform to assess the genomic risk of multiple cancers - towards the precision prevention of cancer





.project-pillars-left[
### Project Pillars
* Flexible Genotyping Platform
    
* Standard guidelines for interpretation

* Integrating PRS into the existing risk management landscape
]

.project-pillars-right[
&lt;img src="presentation_files/figure-html/pillars-1.png" width="100%" /&gt;
]

---
class: inverse

# VCCC Precision Medicine Grant: Sequencing 
## Developing an optimal Genomic Risk platform

.platform-left[

### 1. Establish a research platform 
Custom Illumina Microarray Chip 
- 270,000 SNP Core

Plus 600,000 SNPs identified by OncoArray Network contributors: 

&lt;ul&gt;
    &lt;li title="Genetic Associations and Mechanisms in Oncology"&gt;GAME-ON&lt;/li&gt;
    &lt;li title="Breast Cancer Association Consortium"&gt;BCAC&lt;/li&gt;
    &lt;li title="Ovarian Cancer Association Consortium"&gt;OCAC&lt;/li&gt;
    &lt;li title="Consortium of Investigators of Modifiers of BRCA1/2"&gt;CIMBA&lt;/li&gt;
&lt;/ul&gt;

.img-infinium[![](static/images/infinium.png)]

]


.platform-right[

### 2. Assess requirements for clinical accreditation 

![](static/images/nata.jpg)
]

---

class: inverse

# VCCC Precision Medicine Grant: Interpretation
## Standardising Polygenic Risk Assessment

.mcmahon-left[
.mcmahon-meme[
![](static/images/mcmahon-1.jfif)
&lt;p&gt;BRCA Plus Panel&lt;/p&gt;&lt;br /&gt;
]
]

.mcmahon-right[
.mcmahon-meme[
![](static/images/mcmahon-2.jfif)
&lt;p&gt;62 SNP iCOGS Breast Score (2015)&lt;sup&gt;1&lt;/sup&gt;&lt;/p&gt;
&lt;p&gt;0.619&amp;lt;AUC&amp;lt;0.627&lt;/p&gt;
]
]

.mcmahon-left[
.mcmahon-meme[
![](static/images/mcmahon-3.jfif)
&lt;p&gt;313 SNP Score (2018)&lt;sup&gt;2&lt;/sup&gt;&lt;/p&gt;
&lt;p&gt;0.628&amp;lt;AUC&amp;lt;0.651&lt;/p&gt;
]
]

.mcmahon-right[
.mcmahon-meme[
![](static/images/mcmahon-4.jfif)
&lt;p&gt;Clinically valid multi-cancer PRS informing management of high-risk variants &amp; lifestyle factors (&lt;em&gt;2023?&lt;/em&gt;)&lt;/p&gt;
]
]

---

class: inverse

# VCCC Precision Medicine Grant: Reporting
## Using PRS in existing risk assessment processes

---

# A bit about me...

.me-right[
.mcmahon-meme[![](static/images/JM.jfif)]
]

.me-left[
Background:
* BSc (Genetics, Microbiology)
* MPH (Epidemiology, Biostats, Global Health)
* ?PhD 

||
-|-
Self-taught Programmer | ![](static)/images/python.png) | ![](static/images/R_logo.png)
Thinks he can do Data Science |  ![](static)/images/dsc.jfif) | .eqn[ `$$p(X)=\beta\_{0}+\beta\_{1}X$$`]
Trying to be a Web Designer | ![](static)/images/django.png) | ![](static/images/web.jfif)

]

---

class: inverse

# Genome Wide Association Catalog - Cancer

---

# References

.footnote[
[1] 62 SNPs

[2] Mavaddat et al. Polygenic Risk Scores for Prediction of Breast Cancer and Breast Cancer Subtypes. AJHG. 2019;104(1):21â€“34. Available from: https://doi.org/https://doi.org/10.1016/j.ajhg.2018.11.002

[3]

[4]

[5]

[6]
]
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="static/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
